processing/data_collection/gazette/spiders/go_aparecida_de_goiania.py

```diff
@@ -0,0 +1,37 @@
+import datetime as dt
+from json import loads as json_load
+
+import scrapy
+from dateparser import parse
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class GoAparecidaDeGoianiaSpider(BaseGazetteSpider):
+    TERRITORY_ID = "5201405"
+    name = "go_aparecida_de_goiania"
+    allowed_domains = ["aparecida.go.gov.br"]
+    start_urls = ["https://webio.aparecida.go.gov.br/api/diof/lista"]
+
+    def parse(self, response):
+        download_url = 'https://webio.aparecida.go.gov.br/diariooficial/download/{}'
+
+        items = []
```

Comment:
Remove this as you will be yielding the items inside the loop. (see next comment).